lib/persisted-model.js

```diff
@@ -741,13 +784,19 @@ module.exports = function(registry) {
       http: { verb: 'get', path: '/count' },
     });
 
-    setRemoting(PersistedModel.prototype, 'updateAttributes', {
+    var updateAttributesOptions = {
       description: 'Update attributes for a model instance and persist it into the data source.',
       accessType: 'WRITE',
       accepts: { arg: 'data', type: 'object', http: { source: 'body' }, description: 'An object of model property name/value pairs' },
       returns: { arg: 'data', type: typeName, root: true },
-      http: { verb: 'put', path: '/' },
-    });
+      http: [{ verb: 'patch', path: '/' }],
+    };
+
+    setRemoting(PersistedModel.prototype, 'updateAttributes', updateAttributesOptions);
```

Comment:
I am proposing to rename this method to `patchAttributes` and move `updateAttributes` to the list of aliases.

In 2.x, we should preserve `updateAttributes` and set `pachAttributes` as an alias only.